ostc4pack/create_full_update_bin.sh

```
@@ -81,18 +95,25 @@ fi
 mkdir -p ./$BUILD_TYPE
 cd ./$BUILD_TYPE
 
-pushd $BUILD_PATH/$CPU1_DISCOVERY/$BUILD_TYPE/
-$PACKAGE_TOOL_DIR/OSTC4pack_V4 1 ${PROJECT_NAME_PREFIX}${CPU1_DISCOVERY}.bin
-CHECKSUM_COMMAND_PARAMETERS="${CHECKSUM_COMMAND_PARAMETERS} $(pwd)/${PROJECT_NAME_PREFIX}${CPU1_DISCOVERY}_upload.bin"
-popd
+if [ -z "${DO_BOOTLOADER:+x}" ]; then
+    pushd $BUILD_PATH/$CPU1_DISCOVERY/$BUILD_TYPE/
```

Shellcheck (0.11.0)

[warning] Use 'pushd ... || exit' or 'pushd ... || return' in case pushd fails.

(SC2164)